lldb-eval/eval.h

```diff
@@ -20,7 +20,6 @@
 #include "clang/Basic/TokenKinds.h"
 #include "expression_context.h"
 #include "lldb-eval/ast.h"
-#include "lldb-eval/defines.h"
```

Comment:
Add this include to all files that use `lldb_eval_unreachable`, please. Ideally every symbol usage should have an appropriate include for it (it may not be the case currently, but we try to adhere to this rule).